plugins/ai.js

```
@@ -3,184 +3,184 @@
-		).BK9;
-		return await msg.edit(res);
-	}
+  {
+    pattern: 'ai',
+    public: true,
+    desc: 'Chat with an AI Bot',
+    type: 'ai',
+  },
+  async (message, match, { pushName }) => {
+    if (!match && !message.reply_message.text)
+      return message.send('_Hello there ' + pushName + '_');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)